### ISSUE TITLE
Errors in ./run but ldapd seems started

### ISSUE THREAD

**Issue by jrevillard** [tiredofit/docker-openldap#14]
When the openldap starts, I get those errors in the logs:

```
openldap-fusiondirectory_openldap-fusiondirectory.1.lp8ifqeq7h9a@swarm2-node2.gnubila.fr    | [INFO] ** [openldap] Starting OpenLDAP 
openldap-fusiondirectory_openldap-fusiondirectory.1.lp8ifqeq7h9a@swarm2-node2.gnubila.fr    | ./run: line 17: unexpected EOF while looking for matching `"'
openldap-fusiondirectory_openldap-fusiondirectory.1.lp8ifqeq7h9a@swarm2-node2.gnubila.fr    | ./run: line 19: syntax error: unexpected end of file
openldap-fusiondirectory_openldap-fusiondirectory.1.lp8ifqeq7h9a@swarm2-node2.gnubila.fr    | 5e0dcb45 @(#) $OpenLDAP: slapd 2.4.48 (Jan  1 2020 04:50:13) $
openldap-fusiondirectory_openldap-fusiondirectory.1.lp8ifqeq7h9a@swarm2-node2.gnubila.fr    | 	@48f7866aff2f:/tiredofit/openldap:6.6.1/servers/slapd
openldap-fusiondirectory_openldap-fusiondirectory.1.lp8ifqeq7h9a@swarm2-node2.gnubila.fr    | ./run: line 17: unexpected EOF while looking for matching `"'
openldap-fusiondirectory_openldap-fusiondirectory.1.lp8ifqeq7h9a@swarm2-node2.gnubila.fr    | ./run: line 19: syntax error: unexpected end of file
openldap-fusiondirectory_openldap-fusiondirectory.1.lp8ifqeq7h9a@swarm2-node2.gnubila.fr    | 5e0dcb45 slapd starting
```

Is it a configuration issue on my side ?
Best,
Jerome

**Comment by tiredofit** [comment]
Thanks for catching this - This was related to the nginx script starting and a misplaced quote. Fixed in `6.6.2`

**Comment by jrevillard** [comment]
It works thx!
